src/main/resources/ListedElectronicMonograph.json

```diff
@@ -111,6 +104,7 @@
           "field": "LOCATION",
           "dataSource": {
             "from": "//Location",
+            "default": "*",
```

Comment:
is this default ever used?  Maybe I'm reading the code wrong, but it looks like it's hardcoded in PostGobiOrdersHelper.  If it is used, I think you probably need `"translateDefault": true`

Reply:
Also applies to other mapping files...

Reply:
Based on the unit tests, I must be missing something because it does look like this is being used and not hardcoded... I still think the translateDefault part is required.

Reply:
That is true, translateDefault is required. Also I also corrected another problem for a scenario where the /locations or /material-types endpoints return no entries(for a case where the reference data is not loaded).